docs/Fields.md

```diff
@@ -520,6 +520,15 @@ You can also prevent `<ReferenceField>` from adding link to children by setting
 </ReferenceField>
 ```
 
+You can also use a custom `linkType` function to get a custom path for the children.
+
+```jsx
+// Custom path
+<ReferenceField label="User" source="userId" reference="users" linkType={() => '/my/path'}>
```

Comment:
please use a more meaningful example, e.g. by redirecting to a different path depending on the reference record.